streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/LogClientService.java

```diff
@@ -33,10 +33,10 @@
 @Slf4j
 @Component
 public class LogClientService {
-  public String rollViewLog(String path, int skipLineNum, int limit) {
+  public String rollViewLog(String path, int offset, int limit) {
     String result = "";
     try {
-      List<String> lines = readPartFileContent(path, skipLineNum, limit);
+      List<String> lines = readPartFileContent(path, offset, limit);
```

Comment:
The result can be removed, and return `""` in the end of this method directly.

Reply:
> The result can be removed, and return `""` in the end of this method directly.

thanks for your review, done.